lib/edoc/src/edoc_extract.erl

```diff
@@ -230,6 +230,14 @@ add_macro_defs(Defs0, Opts, Env) ->
 
 %% INHERIT-OPTIONS: text/4
 
+-spec file(File, Context, Env, Options) -> {ok, Tags} | {error, Reason} when
+      File :: filename(),
+      Context :: overview,
```

Comment:
Should probably be <code>Context :: overview | module</code>. It is a bit difficult to see if it is sufficient or not, but these are the two values I can see are used. Maybe it should also contain <code>footer | function | single</code> (see edoc_tags:tags/0), but I'm not really sure about that...